io/aio-wrapper.cpp

```diff
@@ -376,7 +376,7 @@ namespace photon
         if (ctx->evfd < 0)
             LOG_ERRNO_RETURN(0, -1, "failed to create eventfd");
 
-        int ret = io_setup(IODEPTH, &ctx->aio_ctx);
+        int ret = io_setup(iodepth < IODEPTH ? iodepth : IODEPTH, &ctx->aio_ctx);
```

Comment:
no need for IODEPTH?

Reply:
IODEPTH is now the maximal value of iodepth, since the stack allocated event array set to fixed size of IODEPTH.

It could rename to MAX_IODEPTH